workflows_core/dataset/dataset.py

```diff
@@ -132,3 +188,25 @@ def update_metadata(self, metadata: Dict[str, Any]):
 
     def get_metadata(self) -> Dict[str, Any]:
         return self._api._get_metadata(dataset_id=self._dataset_id)
+
+    def insert_medias(
```

Comment:
I think this should be that this is referring to **local** media files and not just any "medias". (for example those in Memory would not count)